kubemarine/procedures/upgrade.py

```diff
@@ -97,6 +104,14 @@ def upgrade_packages(cluster: KubernetesCluster):
 def upgrade_plugins(cluster):
     upgrade_version = cluster.context["upgrade_version"]
 
+    # Procedure for removing outdated jobs for nginx version 1.4.0 and older
+    version_nginx = cluster.inventory['plugins']['nginx-ingress-controller']['version']
+    minor_version = int(version_nginx.split('.')[1])
+    if minor_version >= 7:
+        first_control_plane = cluster.nodes["control-plane"].get_first_member()
+        first_control_plane.sudo(f"sudo kubectl delete job ingress-nginx-admission-create -n ingress-nginx && "
+                                 f"sudo kubectl delete job ingress-nginx-admission-patch -n ingress-nginx")
```

Comment:
As discussed, it is not a problem of upgrade, but a problem of reinstallation. So we need to fix the reinstallation procedure instead.